modules/microshift-greenboot-how-workload-health-check-scripts-work.adoc

```diff
@@ -0,0 +1,26 @@
+//Module included in the following assemblies:
+//
+//* microshift_running-applications/microshift-greenboot-workload-scripts.adoc
+
+:_content-type: CONCEPT
+[id="microshift-greenboot-how-workload-health-check-scripts-work_{context}"]
+= How workload health check scripts work
+
+The workload or application health check script described in this tutorial uses the {product-title} health check functions that are available in the `/usr/share/microshift/functions/greenboot.sh` file. This enables you to reuse procedures already implemented for the {product-title} core services.
+
+[NOTE]
+====
+The provided API can change from release to release. After upgrading {product-title}, check your scripts against the most recent version.
```

Comment:
@ggiguash I'm still a little worried about this. I don't know if we're going to be able to get away with saying this integration point does not have a stable API if we document using it here.

Reply:
Those look like features we can support long term. If we want to do that, we should remove the disclaimer.

Reply:
I'm OK for follow your recommendation on this, @dhellmann. It was originally without the disclaimer, so I'm fine removing it.